### PR TITLE
add bungalows 21 and 22

### PR DESCRIPTION
**Pull request by LilShiningMan**
#### Purpose of change

increase house spawn variety

#### Testing
linted and spawned in-game

#### Additional context
bungalow21 g
bungalow21 r
bungalow22 g
bungalow22r

### CODE REVIEW

**Comment by Firestorm01X2** [comment]
изображение
You have conflicts in PR. It must be resolved before PR can be accepted.

Additionally I recommend to update your orging repository from time to time.

File: data/json/overmap/multitile_city_buildings.json [hunk 1]
```diff
@@ -209,6 +209,42 @@
...
+  },
+  {
+    "type": "city_building",
+    "id": "bungalow20",
+    "locations": [ "land" ],
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "bungalow20_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "bungalow20_roof_north" }
+    ]
+  },
```
Comment (scarf005) [inline data/json/overmap/multitile_city_buildings.json:229]: ## Why a merge conflict is happening?
- PR has 4 changes: bungalows(B) 19, 20, 21, 22. 
- main branch already have B19, B20 merged via #2433 

So to git, this PR tries to overwrite B19, B20. So git asks the person whether to overwrite or not (so you don't overwrite important things). this is called **Merge Conflict**.

## How do i fix it?

github has a [doc for resolving conflict](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) via online editor.

If you're comfortable with git, i'll recommend completely removing B19 and B20 from json then git amend and force push.

**Review by scarf005 — CHANGES_REQUESTED** [review]

**Review by scarf005 — APPROVED** [review]
I've used [github web merge conflict editor](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to fix the conflicts. 

|   21(1F)    |   21(2F)    |
| :---------: | :---------: |
| ![][21(1f)] | ![][21(2f)] |
|   22(1F)    |   22(2F)    |
| ![][22(1f)] | ![][22(2f)] |

overrall, looks nice. also, may i ask which tool do you use to write buildings?

[21(2f)]: https://user-images.githubusercontent.com/54838975/226499441-4967e8b5-0cfc-46fb-a1d3-1a15421eac6b.png
[21(1f)]: https://user-images.githubusercontent.com/54838975/226499447-03282d88-1c3f-42c4-a3f5-bc3d544171eb.png
[22(1f)]: https://user-images.githubusercontent.com/54838975/226499449-30a1d068-247e-419d-a140-f3a42a473700.png
[22(2f)]: https://user-images.githubusercontent.com/54838975/226499450-60f12bbe-6fdb-4823-a5b8-3f1a4a3715ac.png

**Comment by LilShiningMan** [comment]
> @LilShiningMan If you have any questions about resolving merge conflicts -feel free to ask.

Thanks Firestorm. I'm usually pretty good at updating my repository before making PRs, but obviously didn't this time. Github hates me, so if I need additional help I'll let you know.

==========

> I've used [github web merge conflict editor](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to fix the conflicts.
> 21(1F) 	21(2F)

> overrall, looks nice. also, may i ask which tool do you use to write buildings?

great, thanks scarf, didn't even know about that tool so I'll keep that in mind.  

As for what tools I use, I made my own with spreadsheets years ago -- it works for me, and I know them really well and can make maps pretty quickly, but if I were starting from scratch I'd probably start using the [Hostile Architect tool](https://discourse.cataclysmdda.org/t/hostile-architect-location-builder-tool-extra-locations-mod/26351). If you're interested, you can check out the thread -- I have some comments on there where I have screenshots of the spreadsheets I made.

File: data/json/mapgen/house/bungalow21.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      "palettes": [ "standard_domestic_palette", "standard_domestic_lino_bathroom", "standard_domestic_landscaping_palette" ],
+      "fill_ter": "t_floor",
+      "rows": [
+        ".[...........p._______..",
+        ".........??.---_______..",
+        "......[..??.---_______/.",
+        "............--#]]]]]]]#.",
+        "...%###oo###o*#,,,,,,,#.",
+        "...%#@@  dd|L |,,,,,,,#.",
+        "...%o≠     |$ |,,,,,,,#.",
```
[the remark this answers is not in the document]
Comment (LilShiningMan) [reply data/json/mapgen/house/bungalow21.json]: oops. This was previously merged in CDDA, and there was a toybox there. I don't think that exists in BN, so I've taken it out

**Review by scarf005 — APPROVED** [review]
tests pass and all looks good.

**Pull request merged**